cpp/lib/src/ProcessUtil.cc

```diff
@@ -0,0 +1,52 @@
+/** \file    ProcessUtil.c
+ *  \brief   The ProcessUtil implementation.
+ *  \author  Dr. Johannes Ruscheinski
+ */
+
+/*
+ *  Copyright 2020 Universitätsbibliothek Tübingen
+ *
+ *  This file is part of the libiViaCore package.
```

Comment:
Also the whole libiViaCore stuff

Reply:
By the way, do we really need a license statement in each file? In the meantime it is possible to set a license for a github repository.

Reply:
Well, maybe we don't